openlibrary/plugins/worksearch/autocomplete.py

```diff
@@ -45,7 +45,7 @@ def doc_filter(self, doc: dict) -> bool:
     def GET(self):
         return self.direct_get()
 
-    def direct_get(self, fq: list[str] | None = None):
+    def direct_get(self, fq: tuple[str, ...] | None = None):
```

Comment:
This is better as an iterable; that'll let the method support lists as well (in case other spots are specifying lists instead of tuples).

```suggestion
    def direct_get(self, fq: Iterable[str] | None = None):
```

Reply:
Yeah, it worked and no issues occurred.